pkg/cloudprovider/aws/launchtemplate.go

```diff
@@ -192,26 +193,51 @@ func (p *LaunchTemplateProvider) getSecurityGroupIds(ctx context.Context, provis
 
 func (p *LaunchTemplateProvider) getUserData(ctx context.Context, provisioner *v1alpha3.Provisioner, constraints *Constraints) (string, error) {
 	var userData bytes.Buffer
-	userData.WriteString(fmt.Sprintf("[settings.kubernetes]\napi-server = \"%s\"\n", provisioner.Spec.Cluster.Endpoint))
-	userData.WriteString(fmt.Sprintf("cluster-name = \"%s\"\n", *provisioner.Spec.Cluster.Name))
+	userData.WriteString(fmt.Sprintf(`#!/bin/bash
+yum install -y https://s3.amazonaws.com/ec2-downloads-windows/SSMAgent/latest/linux_amd64/amazon-ssm-agent.rpm
+/etc/eks/bootstrap.sh '%s' \
```

Comment:
does this bootstrap script mean that the userdata only works with EKS or will this work fine with kops and other control-plane deployments?

Reply:
I had the same question - according to @ellistarn the script, despite the name, is not EKS-specific.

Reply:
ok that's good, I guess we'll see. I'm about to set this up in kops